terraform/modules/website-e2e-test/build.tf

```diff
@@ -0,0 +1,32 @@
+# Build & Store Container Images
+
+# TODO: Terraform will always think these resources will change due to 
+# the filename parameter, which is not required, but still populated by
+# the API.  Investigate work-around.
```

Comment:
See #718 

This can be resolved by removing the `git_file_souce` block and instead adding a `filename` parameter instead.

Reply:
See https://github.com/GoogleCloudPlatform/emblem/pull/723/commits/a20622986864fb81f20a7a6e49b4904b7bd128c2

Reply:
Done.